src/test/java/org/folio/rest/impl/InvoicesApiTest.java

```diff
@@ -480,6 +486,60 @@ public void testTransitionToApprovedWithoutInvoiceLines() {
 
   }
 
+  @Test
+  public void testTransitionToApprovedWithFundDistributionsEmpty() {
+    logger.info("=== Test transition invoice to Approved with Fund Distributions empty will fail ===");
+
+    InvoiceLine invoiceLine = getMockAsJson(INVOICE_LINE_SAMPLE_PATH).mapTo(InvoiceLine.class);
+    Invoice reqData = getMockAsJson(REVIEWED_INVOICE_WITH_EXISTING_VOUCHER_SAMPLE_PATH).mapTo(Invoice.class);
+    reqData.setStatus(Invoice.Status.APPROVED);
+    String id = reqData.getId();
+    invoiceLine.setId(UUID.randomUUID().toString());
+    invoiceLine.setInvoiceId(id);
+    invoiceLine.setFundDistributions(null);
```

Comment:
Changes in the scheme work if you specify null for `fundDistributions`.
But if you do not specify `fundDistributions `in the request, RMB will convert it into an empty array, which will cause 422 http errors because there is restriction `"minItems": 1` in [invoice_line.json](https://github.com/folio-org/acq-models/blob/1e3802b5a0db84a111dfe347f75c7cfde5642882/mod-invoice-storage/schemas/invoice_line.json#L39) shema

Reply:
removed schema restriction